zkevm/tests/integration.rs

```diff
@@ -21,19 +21,19 @@ fn test_load_params() {
     init();
     log::info!("start");
     load_params(
-        "/home/ubuntu/scroll-zkevm/zkevm/test_params",
+        "/home/ubuntu/scroll-zkevm/zkevm/kzg_params",
```

Comment:
Sorry, but is this the right path? scroll-zkevm?

Reply:
I think this path should be updated as local user's directory environment. 
I changed the last directory name for default directory name

Reply:
Oh it is ignored so it was fine. Maybe let's fix the path later when enabling it.